zulipterminal/model.py

```diff
@@ -96,17 +113,17 @@ def __init__(self, controller: Any) -> None:
                 ('subscription', self._handle_subscription_event),
                 ('typing', self._handle_typing_event),
                 ('update_message_flags',
-                 self._handle_update_message_flags_event),
+                    self._handle_update_message_flags_event),
                 ('update_display_settings',
-                 self._handle_update_display_settings_event),
+                    self._handle_update_display_settings_event),
```

Comment:
I guess these diffs aren't needed. :)

Reply:
Don't remember messing with that, I'll remove them